lhs.js

```diff
@@ -117,13 +144,10 @@ class DomLhs extends Lhs {
     }
 
     fnodes(ruleset) {
-        const matches = ruleset.doc.querySelectorAll(this.selector);
         const ret = [];
-        for (let i = 0; i < matches.length; i++) {  // matches is a NodeList, which doesn't conform to iterator protocol
-            const element = matches[i];
-            ret.push(ruleset.fnodeForElement(element));
-        }
-        return ret;
+        const matches = ruleset.doc.querySelectorAll(this.selector);
+        Array.from(matches).forEach(element => ret.push(ruleset.fnodeForElement(element)));
```

Comment:
There's no need to use Array.from(). NodeLists have their own forEach() method.

Reply:
For some reason, it's not able to find that method. I'm not sure what the issue could be, I'm just using matches.forEach(blah), and it gives me "TypeError: matches.forEach is not a function".

Reply:
jsdom must not have implemented it. Makes me want to go back to the `for` loop.